homeassistant/components/assist_pipeline/__init__.py

```diff
@@ -58,6 +58,7 @@ async def async_pipeline_from_audio_stream(
     conversation_id: str | None = None,
     tts_audio_output: str | None = None,
     device_id: str | None = None,
+    detect_wake_word: bool = False,
```

Comment:
I would suggest we allow passing in `start_stage`, like we do for the other APIs.